extensions/dicom-seg/src/components/SegmentationSettings.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import { Range } from '@ohif/ui';
+
+import './SegmentationSettings.css';
+
+const SegmentationSettings = ({ configuration, onBack, onChange }) => {
+  const [state, setState] = useState({ ...configuration });
+
+  useEffect(() => {
+    setState(state => ({ ...state, ...configuration }));
+  }, [configuration]);
+
+  useEffect(() => {
+    onChange(state);
+  }, [state]);
```

Comment:
If this updates configuration in the parent, which updates local state, I'm not sure why we have local state and merge above.

Reply:
Local state is to have this component independent of the parent, this abstraction is to have it like a reusable form, given defaults, set local state and propagate current state on change. That merge is unnecessary though, updated.